python/cucim/src/cucim/skimage/measure/_colocalization.py

```diff
@@ -261,7 +269,15 @@ def manders_overlap_coeff(image0, image1, mask=None):
     if image1.min() < 0:
         raise ValueError("image1 contains negative values")
 
-    return _get_manders_overlap_coeff(image0, image1)
+    try:
+        return _get_manders_overlap_coeff(image0, image1)
+    except driver.CUDADriverError:
+        # One case on CI recently failed to fuse kernels. I was unable to
+        # reproduce this locally with the same NumPy and CuPy versions.
+        warnings.warn(
+            "kernel fusion failed, falling back to non-fused implementation"
+        )
+        return _get_manders_overlap_coeff_no_fusion(image0, image1)
```

Comment:
cc @leofang (in case you have insight into what causes this to fail on CUDA 11.2)

Reply:
What's the GPU & driver version in use, do you know @jakirkham ?

Reply:
From [one CI log]( https://github.com/rapidsai/cucim/actions/runs/5524163077/jobs/10078441622?pr=572#step:7:710 ) showing this issue, am seeing the following:

<details>

```
Tue Jul 11 21:46:39 2023       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.182.03   Driver Version: 470.182.03   CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Tesla V100-SXM2...  On   | 00000000:0A:00.0 Off |                    0 |
| N/A   34C    P0    42W / 300W |      0MiB / 32510MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
```

</details>